Makefile

```diff
@@ -137,7 +137,7 @@ $(CODE_GENERATOR):
 	GOBIN=$(TOOLS_GOBIN_DIR) $(GO_INSTALL) github.com/kcp-dev/code-generator $(CODE_GENERATOR_BIN) $(CODE_GENERATOR_VER)
 
 lint: $(GOLANGCI_LINT) $(STATICCHECK) $(LOGCHECK)
-	$(GOLANGCI_LINT) run --timeout=10m ./...
+	$(GOLANGCI_LINT) run ./...
```

Comment:
is removing the timeout safe? I guess so but wondering why it was there in the first place.

Reply:
IIRC on the GitHub actions bits (or Prow?) things don't end up with great DX on timeout of the action itself or something and it's much nicer to have the runner timeout and exit

Reply:
This got actually moved to the `.golangci-lint` file

Reply:
ack 👍